src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -819,6 +822,64 @@ export default {
       }
     },
 
+    /** Loads A Transition filing into the historyItems list. */
+    loadTransitionFiling (filing: FilingIF) {
+      const header = filing?.header
+      const business = filing?.business
+
+      if (header && business) {
+        const filingType = FilingTypes.TRANSITION_APPLICATION
+
+        let subtitle = ''
+
+        const filingDateTime = this.convertUTCTimeToLocalTime(header.date)
+        const filingDate = filingDateTime?.slice(0, 10)
+
+        // Effective Date is assigned by the backend when the filing is completed (normally right away).
+        // Effective Date may be in the future (eg, for BCOMP COA filings).
```

Comment:
You are right to handle FE date here, but please update the comment here.

Also please update the variable names on lines 844 and 847 (and comments).

Reply:
Haha, those variables aren't even being used.
Anyway thanks for the update.